### PR TITLE
f-scoring-per-category

### PR DESCRIPTION
**Pull request by francescajs**
score input section for every category to be scored
put to database
reflected in the rendered list in scoring overview

### CODE REVIEW

File: client/src/components/ProjectInfo/PerCategory.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+// import './PerCategory.css';
```
Comment (lajiang2) [inline client/src/components/ProjectInfo/PerCategory.jsx]: comment

File: client/src/components/ProjectInfo/ProjectInfo.jsx [hunk 2]
```diff
@@ -12,54 +13,79 @@ class ProjectInfo extends Component{
         this.state = {
             judgeId: '',
             projectId: '',
-            score:''
+            scores:{}
```
Comment (lajiang2) [inline client/src/components/ProjectInfo/ProjectInfo.jsx]: missing a space

File: client/src/components/ScoringOverview/Project.jsx [hunk 3]
```diff
@@ -10,8 +10,8 @@ class Project extends Component {
                     <h3>{this.props.name}</h3>
                     <p>ID: {this.props.identification}</p>
                 </div>
-                <div className="overview-score">
-                    <h2>{this.props.score}</h2></div>
+                {/* <div className="overview-score">
```
Comment (lajiang2) [inline client/src/components/ScoringOverview/Project.jsx]: comment

File: client/src/components/ScoringOverview/ScoringOverview.jsx [hunk 4]
```diff
@@ -21,7 +21,7 @@ class ScoringOverview extends Component {
     }
 
     async componentDidMount() {
-        const res = await fetch('/api/projects/' + this.props.location.state.judgeId + '/');
+        const res = await fetch(`/api/toscore/judge/` + this.props.location.state.judgeId);
```
Comment (lajiang2) [inline client/src/components/ScoringOverview/ScoringOverview.jsx]: use `'` not `\``

**Review by lajiang2 — CHANGES_REQUESTED** [review]

**Review by lajiang2 — APPROVED** [review]

**Review by partyshah — APPROVED** [review]

**Pull request merged**